api/ShadowRoot.json

```diff
@@ -494,6 +494,53 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "accepts_TrustedHtml": {
+          "__compat": {
+            "description": "Can be set with a `TrustedHtml`",
```

Comment:
I would think "Can be set with a `TrustedHtml`" is ungrammatical and should be "Can be set with a `TrustedHtml` instance"

Reply:
```suggestion
            "description": "Can be set with a `TrustedHtml` instance",
```